timemachine/fe/rbfe.py

```diff
@@ -116,7 +116,7 @@ def sample(initial_state, protocol):
         n_steps=n_steps,
         lambda_windows=[initial_state.lamb],
         store_u_interval=protocol.steps_per_frame,
-        store_x_interval=protocol.steps_per_frame,
+        store_x_interval=int(protocol.steps_per_frame / protocol.subsample_traj),
```

Comment:
1) I think this also affects the # of samples used in the free energy estimate.
2) Diving by subsample_traj would *increase* number of frames we'd get.  (eg `subsample_traj=2`, `steps_per_frame=100` would result a `store_x_interval` of `50`


I'd rather subsampling of frames be done *outside* of the estimator, and it should only affect the storage component. It might be easier to instead do `simulation_result.frames[::subsample_traj]` instead.

Reply:
good point